### ISSUE TITLE
[Community Bounty] AugurInsider

### ISSUE THREAD

**Issue by SebastianMetzger** [AugurProject/augur#1247]
## Milestone 1: Reanimate AugurInsider and intergrate scalar markets
Timeframe: Until April 30th
## Milestone 2: Update UI and integrate historic values and charts of AIVIX and other KPIs
Timeframe: Until June 30th
### ETH Address
- TBD -

**Comment by SebastianMetzger** [comment]
**Update 2019-03-12**
Had a small session over the weekend with two colleagues, one a mathematician and the other one a designer. Going to work on this together 👍

**Comment by SebastianMetzger** [comment]
**Update 2019-03-13**
"Resurrected" the page by adding current markets, some calculation issues still have to be solved: https://augurinsider.com/

**Comment by SebastianMetzger** [comment]
Just had another session and already fixed some stuff in the black scholes calculations so everything is returned now and there are no more NaN results.
https://augurinsider.com/

Also we made a concept for the UI with graphs and historic data. 👍

**Comment by petervecchiarelli** [comment]
Hey Sebastian, hope everything's going well. Just a friendly reminder to check in when you've completed milestone 1!

**Comment by SebastianMetzger** [comment]
We just updated the site and the veil markets are now included besides some other small view changes, unfortunately the volume/liquidity in those markets is also almost zero, which makes what we do a bit useless.
https://augurinsider.com/

Maybe we will pivot a bit in a direction that will maybe create some liquidity in the short term, so in the long term anylsis and KPIs that we want to provide will make more sense. Will do another planning session on the weekend :)

**Comment by SebastianMetzger** [comment]
@petervecchiarelli @tomkysar 
While working on milestone 1 we realized volume and liquidity in the currency derivative markets neither on veil nor on Augur took off as we hoped.
With the current data basis the original vision of augurinsider in providing trading KPIs and volatitly indexes simply does not make sense yet.

We are still excited about the use case of using Augur to insure and hedge against price fluctuations in digital currencies, but currently there is no project that really creates an easy to understand user experience around this.

So we decided to pivot our project in that direction and first get more users to actually use derivative markets, before we go back to build out KPIs and metrics surrounding all this.
More details will follow, but it will go in the direction of having a really simple UI, that translates whats happening in the background into terms of a simple insurance everybody can understand.
Probably we will open source this also from the beginning.

It is up to you how this will affect the community grant and we will work on this either way, but in our eyes getting more people into the markets is currently more important than creating analysis tools.
